src/lib/markbind/src/parser.js

```diff
@@ -392,13 +397,18 @@ class Parser {
         const userDefinedVariables = config.userDefinedVariablesMap[path.resolve(parent, relative)];
         const pageVariables = this.extractPageVariables(file, data, userDefinedVariables, {});
         let fileContent
-          = nunjucks.renderString(
-            data,
-            { ...pageVariables, ...userDefinedVariables },
-            { path: actualFilePath });
+          = nunjuckUtils.renderEscaped(nunjucks,
+                                       data,
+                                       {
+                                         ...pageVariables,
+                                         ...userDefinedVariables,
+                                         path: actualFilePath,
```

Comment:
`renderEscaped` does not seem to pass the third `options` argument with `path` to `nunjucks.renderString`;
Let's keep the `{ path: ... }` object separate as well since the user might have a variable named `path`